metrics/bigfiles_high_water_mark

```diff
@@ -1 +1 @@
-1049
+1061
```

Comment:
`tests/integration/itest` is large.  At some point it ought to be split up into modules, but I'm not entirely sure it's a net complexity win to do that yet.